### PR TITLE
Update datadog annotations for deployment

### PR DESCRIPTION
**Pull request by andibeuge**
We need to correct the labels for datadog

### CODE REVIEW

File: parcellab/app/templates/deployment.yaml [hunk 1]
```diff
@@ -5,9 +5,11 @@ metadata:
   name: {{ $fullname }}
   labels:
     {{- include "common.labels" . | nindent 4 }}
+    {{- if and .Values.datadog .Values.datadog.enabled }}
     tags.datadoghq.com/env: {{ include "common.env" . }}
     tags.datadoghq.com/service: {{ $fullname }}
     tags.datadoghq.com/version: {{ include "common.version" . }}
+    {{- end }}
```
Comment (darioblanco) [inline parcellab/app/templates/deployment.yaml:12]: Actually these labels might not be needed at all, the ones from the spec are the only ones that will be applied to the pods, isn't it?

**Review by darioblanco — DISMISSED** [review]
Just a tiny comment, if we do not need datadog labels in the deployment, I would remove the top labels. 🚀

**Review by jmpalomares — DISMISSED** [review]

File: parcellab/app/templates/deployment.yaml (hunk 1 above)
Comment (andibeuge) [reply parcellab/app/templates/deployment.yaml]: Yes, according to the docs it is not needed at deployment level

Comment (andibeuge) [reply parcellab/app/templates/deployment.yaml]: removed

**Review by darioblanco — APPROVED** [review]

**Pull request merged**